website/blog/2023-05-21-databend-weekly-94.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "This Week in Databend #94"
+date: 2023-05-21
+slug: 2023-05-21-databend-weekly
+tags: [databend, weekly]
+description: "Stay up to date with the latest weekly developments on Databend!"
+contributors:
+  - name: andylokandy
+  - name: ariesdevil
+  - name: b41sh
+  - name: BohuTANG
+  - name: Chasen-Zhang
+  - name: dantengsky
+  - name: Dousir9
+  - name: drmingdrmer
+  - name: everpcpc
+  - name: flaneur2020
+  - name: gitccl
+  - name: hantmac
+  - name: Jake-00
+  - name: lichuang
+  - name: PsiACE
+  - name: RinChanNOWWW
+  - name: silver-ymz
+  - name: soyeric128
+  - name: sundy-li
+  - name: TCeason
+  - name: wubx
+  - name: youngsofun
+  - name: zhang2014
+  - name: ZhiHanZ
+authors:
+  - name: PsiACE
+    url: https://github.com/psiace
+    image_url: https://github.com/psiace.png
+---
+
+[Databend](https://github.com/datafuselabs/databend) is a modern cloud data warehouse, serving your massive-scale analytics needs at low cost and complexity. Open source alternative to Snowflake. Also available in the cloud: <https://app.databend.com> .
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+### Computed Columns
+
+Computed columns are used to generate data from other columns by defining a scalar expression. There are two types of computed columns: **STORED** and **VIRTUAL**.
+
+A stored computed column computes the data when the row is inserted and stores the result value like other columns. To create a stored computed column, use this SQL syntax:
+
+  ```sql
+  column_name <type> AS (<expr>) STORED
+  ```
+
+On the other hand, a virtual computed column does not store the result value; it is computed at query time. To create a virtual computed column, use this SQL syntax:
```

Comment:
While a virtual computed column is calculated at query time and does not store the result value. To create one, use this SQL syntax: